packages/style-minify/README.md

```diff
@@ -0,0 +1,100 @@
+# Neutrino Style Minify Middleware
+
+`@neutrinojs/style-minify` is Neutrino middleware for minifying styles using
+[`OptimizeCssAssetsPlugin`](https://www.npmjs.com/package/optimize-css-assets-webpack-plugin). This middleware is usually only
+added during production builds.
+
+[![NPM version][npm-image]][npm-url]
+[![NPM downloads][npm-downloads]][npm-url]
+[![Join the Neutrino community on Spectrum][spectrum-image]][spectrum-url]
+
+## Requirements
+
+- Node.js v6.10+
+- Yarn or npm client
+- Neutrino v7
+
+## Installation
+
+`@neutrinojs/style-minify` can be installed via the Yarn or npm clients.
+
+#### Yarn
+
+```bash
+❯ yarn add @neutrinojs/style-minify
+```
+
+#### npm
+
+```bash
+❯ npm install --save @neutrinojs/style-minify
+```
+
+## Usage
+
+`@neutrinojs/style-minify` can be consumed from the Neutrino API, middleware, or presets. Require this package
+and plug it into Neutrino:
+
+```js
+// Using function middleware format
+const styleMinify = require('@neutrinojs/style-minify');
+
+// Use with default options
+neutrino.use(styleMinify);
+
+// Usage showing overriding options
+neutrino.use(styleMinify, {
+  pluginUseId: 'style-minify',
+  plugin: {
+    assetNameRegExp: /\.css$/g,
+    cssProcessorOptions: {},
+    canPrint: true,
+  }
+});
+```
+
+```js
+// Using object or array middleware format
+
+// Use with default options
+module.exports = {
+  use: ['@neutrinojs/style-minify']
+};
+
+// Usage showing overriding options
+module.exports = {
+  use: [
+    ['@neutrinojs/style-minify', {
+    }]
+  ]
+};
+```
+
+The `plugin` property is passed to [optimize-css-assets-webpack-plugin
+](https://github.com/NMFR/optimize-css-assets-webpack-plugin#configuration).
+
+
+## Customization
+
+`@neutrinojs/style-minify` creates some conventions to make overriding the configuration easier once you are ready to
+make changes.
+
+### Plugins
+
+The following is a list of plugins and their identifiers which can be overridden:
+
+| Name | Description | Environments and Commands |
+| --- | --- | --- |
+| `style-minify` | Minifies styles using `OptimizeCssAssetsPlugin`. | all |
```

Comment:
I just realized that we need to update the web preset docs to add these plugins as well.

Reply:
Should already be good to go…web doesn't use them separately, it uses `@neutrinojs/minify` which has them all.

Reply:
You'll notice the minify plugin is out of date:

https://github.com/mozilla-neutrino/neutrino-dev/tree/master/packages/web#plugins

For these core presets, we have listed out all the transitive plugins and loaders.

Reply:
Ah i see

Reply:
Are you doing that in this PR, or a should we file a follow up?

Reply:
I'll do it here.